packages/AsyncLoader/src/Template.unit.mjs

```diff
@@ -0,0 +1,89 @@
+import { dirname } from 'path';
+import { fileURLToPath } from 'url';
+import test from 'ava';
+import getDOM from '../../../src/testHelpers/getDOM.mjs';
+import Template from './Template.mjs';
+
+const setup = async(hideErrors) => {
+    const basePath = dirname(fileURLToPath(new URL(import.meta.url)));
+    return getDOM({ basePath, scripts: [], hideErrors });
+};
+
+
+const createElement = (document, html) => {
+    const container = document.createElement('div');
+    container.innerHTML = html;
+    return container;
+};
+
+test.only('replaces content of container with content of a template', async(t) => {
```

Comment:
`only` weg. Überall.

Reply:
`…with content of the provided template`(nicht a – a wäre ein zufälliges template)